notebooks/2_read_write.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>to_* functions -&gt; to_* methods ? </p><p> </p><p>(not sure we already explained that difference (might come later?))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>